data/json/items/tool/smoking.json

```diff
@@ -58,6 +58,81 @@
     "qualities": [ [ "SMOKE_PIPE", 1 ] ],
     "melee_damage": { "bash": 1 }
   },
+  {
+    "id": "pipe_water",
+    "type": "TOOL",
+    "name": { "str": "water pipe" },
+    "looks_like": "bottle_plastic",
+    "description": "A mass-manufacured water-pipe, otherwise known as a 'bong.' Headshops often insist these are used solely for tobacco, despite their iconic status as cannabis paraphernalia.",
+    "weight": "750 g",
+    "volume": "900 ml",
+    "longest_side": "30 cm",
+    "flags": [ "FRAGILE_MELEE", "NONCONDUCTIVE" ],
+    "price": 2000,
+    "price_postapoc": 100,
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "watertight": true,
+        "rigid": true,
+        "max_contains_volume": "500 ml",
+        "max_item_volume": "500 ml",
+        "max_contains_weight": "1 kg",
+        "moves": 400
+      }
+    ],
+    "to_hit": -2,
+    "material": [ "glass" ],
+    "symbol": ",",
+    "color": "cyan",
+    "qualities": [ [ "SMOKE_PIPE", 1 ], [ "BOIL", 2 ], [ "CONTAIN", 1 ] ],
+    "melee_damage": { "bash": 4 },
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "pipe_water_normal",
+        "name": { "str": "bong" },
+        "description": "A mass-manufacured water-pipe, otherwise known as a 'bong.' This one is made of plain, clear, glass.",
+        "color": "light_gray"
+      },
+      {
+        "id": "pipe_water_cartoon",
+        "name": { "str": "cartoon bong" },
+        "description": "A mass-manufacured water-pipe, otherwise known as a 'bong.' It is covered in copyrighted cartoon characters, all smoking marijuana.",
+        "color": "green"
+      },
+      {
+        "id": "pipe_water_rasta",
+        "name": { "str": "rasta bong" },
+        "description": "A mass-manufacured water-pipe, otherwise known as a 'bong.' It colored with the 'rasta' pattern of red, yellow, and green.",
+        "color": "red"
+      },
+      {
+        "id": "pipe_water_handblown",
+        "name": { "str": "handblown bong" },
+        "description": "A handblown water pipe, also known as as a 'bong.' Lovely crafted by an artisan, a true fusion of aesthetic sensibilities and stoner functionalities ",
+        "color": "green"
+      },
+      {
+        "id": "pipe_water_psychedelic",
+        "name": { "str": "psychedelic bong" },
+        "description": "A mass-manufactured water pipe, also known as a 'bong.' It is covered in stickers depicting 'psychedelic' art.",
+        "color": "blue"
+      },
+      {
+        "id": "pipe_water_frosted",
+        "name": { "str": "frosted bong" },
+        "description": "A mass-manufactured water pipe, also known as a 'bong.' It is made out of frosted glass.",
+        "color": "white"
+      },
+      {
+        "id": "pipe_water_fakegold",
+        "name": { "str": "gold bong" },
+        "description": "A mass-manufactured water pipe, also known as a 'bong.' Though it looks like it's made out of gold, that is in fact just metallic paint.",
```

Comment:
```suggestion
        "description": "A mass-manufactured water pipe, also known as 'bong.'  Though it looks like it's made out of gold, that is, in fact, just metallic paint.",
```

Reply:
"a 'bong' " is the correct way of writing it. "Known as 'bong' " makes it sound like a proper noun, which it is not.